xm-commons-messaging/src/main/java/com/icthh/xm/commons/messaging/communication/CommunicationMessage.java

```diff
@@ -34,5 +38,29 @@ public class CommunicationMessage {
     private List<Receiver> receiver = new ArrayList<>();
     private Sender sender;
     private List<CommunicationRequestCharacteristic> characteristic;
+
+    public CommunicationMessage setTemplateModel(String templateModel) {
+        CommunicationRequestCharacteristic communicationRequestCharacteristic = new CommunicationRequestCharacteristic();
```

Comment:
I see copy/paste, we need method addCharacteristic

Reply:
Extracted to the another method